common/uint160.go

```diff
@@ -12,10 +12,17 @@ import (
 	"github.com/itchyny/base58-go"
 )
 
-const PREFIXCHAR = byte(53)
+// FOOLPROOFPREFIX used for fool-proof prefix
+// base58.BitcoinEncoding[21] = 'N', base58.BitcoinEncoding[18] = 'K'
+// 33 = len(base58.Encode( (2**192).Bytes() )),  192 = 8bit * (UINT160SIZE + SHA256CHKSUM)
+// ((21 * 58**35) + (18 * 58**34) + (21 * 58**33)) >> 192 = 0x02b824
+const FOOLPROOFPREFIX = 0x02b824 + 1 // +1 for avoid affected by lower 192bits shift-add
```

Comment:
What does it mean by "fool proof"? 😂

Reply:
The "fool proof" means prevent mistake😂 Derivative from bulletproof.

Reply:
We probably want to change the word to be less offensive. I suggest "error correction" or "proofread"

Reply:
Never mind, I think it should be ok 😂